src/backend/joanie/edx_imports/tasks/certificates.py

```diff
@@ -169,7 +166,7 @@ def import_certificates_batch(start, stop, dry_run=False):
             )
         )
 
-    import_string = "%s certificates created, %s errors"
+    import_string = "%s-%s/%s %s certificates created, %s skipped, %s errors"
```

Comment:
What do you think to process a percentage ?
`(start / total) * 100` ?

Reply:
Yes, nice idea